tracker/tracker_object.rst

```diff
@@ -451,6 +451,12 @@ Cookies that are used by analytics are first party cookies.
 
     The function that will disable all first party cookies. Existing ones will be deleted in the next page view.
 
+.. function:: enableCookies()
```

Comment:
Please add it in a separate PR

Reply:
Hmm... i popped up when I changed the target branch...

Reply:
Rebase fixed it.